src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: "Manage reliability with service levels"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+  - /docs/1-establish-objectives-baselines-define-team-slos
+  - /docs/1-adopt-easily-establish-objectives-baselines-define-team-slos
+  - /docs/establish-objectives-baselines-define-team-slos
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/service-level-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/slm-implementation-guide/
+  - /docs/using-new-relic/welcome-new-relic/optimize-your-cloud-native-environment/establish-objectives-baselines-define-team-slos
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/slm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/optimize-slm-guide
+freshnessValidatedDate: never
+---
+
+import omSlTheWholeIsGreater from 'images/om-sl_diagram_theWholeIsGreater.webp'
+
+import apmOverviewScreen from 'images/apm_screenshot-full_overview-screen.webp'
+
+import apmServiceBoundary from 'images/apm_screenshot-crop_service-boundary.webp'
+
+import apmLatencyBaseline from 'images/apm_screenshot-full_latency-baseline.webp'
+
+import apmSLStartFromApm from 'images/apm_screenshot-full_sl-start-from-apm.webp'
+
+import apmAddInputServiceLevel from 'images/apm_screenshot-crop_add-input-service-level.webp'
+
+import apmSyntheticsSetup from 'images/apm_screenshot-crop_synthetics-setup.webp'
+
+import queriesnrqlTransactionListNrql from 'images/queries-nrql_screenshot-full_transaction-list-nrql.webp'
+
+import apmCapabilityFilterClause from 'images/apm_screenshot-crop_capability-filter-clause.webp'
+
+
+Within an organization, communication between stakeholders can often prove more difficult than needed. IT departments usually don't speak in terms that the business-focused parts of an organization can understand, and they in turn don't normally communicate in terms that IT teams find useful. To improve reliability, resolving this language barrier helps to prevent issues. Service level management, or the practice of standardizing data into a universal language that all parties can understand, aids in doing just that.
+
+Service level management best applies to the practice of **Uptime, performance and reliability**, but it also applies to the other practices of **Customer experience**, **Innovation and growth**, and **Operational efficiency** ([learn more about these practices](/docs/new-relic-solutions/observability-maturity/introduction)). Regardless of what area you want to improve, using SLM has two major areas of impact: business outcomes and operational outcomes.
+
+<Tabs stacked>
```

Comment:
I'd consider structuring these in a similar way. For example, three bullets for business outcome and then adjusting the operational outcome into three bullets.

I'm not sure re: business outcome if it should be ordered or unordered. Does this list have to happen in this progression? Then numbers are fine. But if they're just a list of ~items that aren't order-specific, I'd use bullet points.